src/main/java/org/GehtSoftHWByAziz/HW2/Reflection/ReflectionTests/AnotherBunchOfTests.java

```diff
@@ -0,0 +1,21 @@
+package org.GehtSoftHWByAziz.HW2.Reflection.ReflectionTests;
+
+import org.GehtSoftHWByAziz.HW2.Reflection.Test;
+
+public class AnotherBunchOfTests {
+  @Test
+  void passedTest1() {
+    assert (2==2);
```

Comment:
`assert`s are not normally used in tests
it's a java keyword that is used when ran with -ea 
i've yet to see a project where they were used